src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    public void doWork() {
+
+    }
+
+    public void stopWork() {
+
+    }

```

Comment:
The methods `doWork` and `stopWork` in the abstract class `Machine` are empty. Consider providing an implementation or making them abstract to force subclasses to provide their own implementations. For example: `public abstract void doWork();`